src/buffer/out/Row.cpp

```diff
@@ -499,6 +739,11 @@ void ROW::_resizeChars(uint16_t colExtEnd, uint16_t chExtBeg, uint16_t chExtEnd,
     }
 }
 
+til::small_rle<TextAttribute, uint16_t, 1>& ROW::Attributes() noexcept

```

Comment:
used?

Reply:
I'm currently using it in a follow-up branch. I can remove it for this PR, but I also feel like it doesn't hurt keeping it in.